src/Context/product-context.js

```diff
@@ -0,0 +1,24 @@
+import { createContext } from "react";
+import { useReducer } from "react";
+import { useContext } from "react";
+
```

Comment:
instead of writing 3 lines you can simply do this in a single line: `import { createContext, useReducer, useContext } from "react";